### DISCUSSION TITLE
Welcome to Azure-DevOps-SFTP-SSH-download Discussions!

### DISCUSSION THREAD

**Discussion by matejskubic** [Adacta/Azure-DevOps-SFTP-SSH-download#6] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by alv1981** [comment]
only download files if I use ** , the filter does not work 

log errors:
Found 77 files at '/' (/)
2022-02-18T11:44:47.5682252Z ##[debug]Filter: '*Guide'
2022-02-18T11:44:47.6037844Z ##[debug]Match files 0 for *Guide
2022-02-18T11:44:48.5187014Z ##[debug]Set ProcessedFilesCount = '0'
